src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -434,10 +434,11 @@ static void uct_rdmacm_cm_handle_error_event(struct rdma_cm_event *event)
             status = UCS_ERR_REJECTED;
         }
 
-        log_level = UCS_LOG_LEVEL_DEBUG;
+        log_level = UCS_LOG_LEVEL_DIAG;
     } else {
         status    = UCS_ERR_IO_ERROR;
-        log_level = UCS_LOG_LEVEL_ERROR;
+        log_level = (event->event == RDMA_CM_EVENT_UNREACHABLE) ?
```

Comment:
pls align status and log level with master branch logic
https://github.com/openucx/ucx/blob/9aead1c9d79d2d9eabceac0647033f8851ac3198/src/uct/ib/rdmacm/rdmacm_cm.c#L627

Reply:
do you mean make same changes in master or set REJECTED log level to DEBUG in this PR?

Reply:
set status and log_level to same values as they are set on master for same rdma_cm event code

Reply:
ok, aligned